cl/search/management/commands/pacer_bulk_fetch.py

```diff
@@ -0,0 +1,385 @@
+                    self.fetches_in_progress.pop(court_id, None)
+
+            # If we had to skip all courts that we tried this round,
+            # add a small delay to avoid hammering the DB
+            if skipped_courts == courts_at_start:
+                time.sleep(self.interval)
+
+    def process_docs_fetched(self):
+        """Apply tasks to process docs that were successfully fetched from PACER."""
+        cached_fetches = cache.get(self.docs_to_process_cache_key(), [])
+        fetch_queues_to_process = [fq_pk for (_, fq_pk) in cached_fetches]
+        fetch_queues = (
+            PacerFetchQueue.objects.filter(pk__in=fetch_queues_to_process)
+            .select_related("recap_document")
+            .only(
+                "pk",
+                "status",
+                "recap_document__id",
+                "recap_document__ocr_status",
+                "recap_document__is_available",
+                "recap_document__filepath_local",
+            )
+        )
+        for fq in fetch_queues:
+            rd = fq.recap_document
+            needs_ocr = rd.needs_extraction
+            has_pdf = rd.filepath_local is not None
+            fetch_was_successful = fq.status == PROCESSING_STATUS.SUCCESSFUL
+            if fetch_was_successful and has_pdf and needs_ocr:
+                self.throttle.maybe_wait()
+                chain(
+                    extract_recap_pdf.si(rd.pk),
+                    mark_fq_successful.si(fq.pk),
+                ).apply_async(queue=self.queue_name)
+
+    def handle_fetch_docs(self):
+        """Run only the fetching stage."""
+        logger.info("Starting fetch stage in pacer_bulk_fetch command.")
+        try:
+            self.set_user(self.options.get("username", "recap"))
+            self.identify_documents()
+            logger.info(
+                f"{self.user} found {len(self.recap_documents)} documents "
+                f"across {len(self.courts_with_docs)} courts."
+            )
+
+            self.fetch_docs_from_pacer()
+
+            logger.info(
+                f"Created {self.total_launched} processing queues for a total "
+                f"of {len(self.recap_documents)} docs found."
+            )
+            logger.info(
+                f"The following PacerFetchQueues were retried too many times: "
+                f"{cache.get(self.timed_out_docs_cache_key(), [])}"
+            )
+        except Exception as e:
+            logger.error(
+                f"Fatal error in fetch stage: {str(e)}", exc_info=True
+            )
+            raise e
+
+    def handle_process_docs(self):
+        """Run only the processing stage."""
+        logger.info("Starting processing stage in pacer_bulk_fetch command.")
+        try:
+            self.process_docs_fetched()
+        except Exception as e:
+            logger.error(
+                f"Fatal error in process stage: {str(e)}", exc_info=True
+            )
+            raise e
```

Comment:
I don't think you're getting much from this method. Suggest merging with process_docs_fetched (or removing it and letting Sentry catch the error).

Reply:
I agree. This method has been removed.